src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -3244,6 +3244,17 @@ void MacroAssembler::vpmullw(XMMRegister dst, XMMRegister nds, Address src, int
   Assembler::vpmullw(dst, nds, src, vector_len);
 }
 
+void MacroAssembler::vpmulld(XMMRegister dst, XMMRegister nds, AddressLiteral src, int vector_len, Register scratch_reg) {
+  // Used in sign-bit flipping with aligned address.
```

Comment:
You could remove the spurious comment here.

Reply:
removed